Tests/UITests/TestApp/HealthKitTestsView.swift

```diff
@@ -14,22 +14,25 @@ import SwiftUI
 struct HealthKitTestsView: View {
     @Environment(HealthKit.self) var healthKitModule
     @Environment(HealthKitStore.self) var healthKitStore
-
     
     var body: some View {
         List {
             AsyncButton("Ask for authorization") {
                 try? await healthKitModule.askForAuthorization()
```

Comment:
You will probably have to add the toShare types here when you request authorization

Reply:
re other comment about accessing healthstore